consensus/ibft/src/main/java/org/hyperledger/besu/consensus/ibft/payload/MessageFactory.java

```diff
@@ -79,7 +79,7 @@ public RoundChange createRoundChange(
   }
 
   private <M extends Payload> SignedData<M> createSignedMessage(final M payload) {
-    final SECPSignature signature = nodeKey.sign(hashForSignature(payload));
+    final SECPSignature signature = nodeKey.sign(qbftHashForSignature(payload));
```

Comment:
Why is qbftHashForSignature being used for ibft?

Reply:
most likely because I made an "oopsie"